scanpipe/models.py

```diff
@@ -1514,6 +1514,14 @@ def get_policies_dict(self):
 
         return scanpipe_app.policies
 
+    def get_license_clarity_compliance_alert(self):
+        """
+        Return the license clarity compliance alert value for the project,
+        or None if not set.
+        """
+        extra_data = self.extra_data or {}
+        return extra_data.get("license_clarity_compliance_alert")
```

Comment:
Are you sure `extra_data = self.extra_data or {}` is required? The default value for this field is already an empty dict. This should be good enough:
```
return self.extra_data.get("license_clarity_compliance_alert")
```